design/crud-api/README.md

```diff
@@ -29,6 +30,63 @@ go test -v ./...
 ./crud-service
 ```
 
+## Database Setup
+
+### PostgreSQL Setup
+
+1. **Using Docker**:
+```bash
+# Run PostgreSQL container
+docker run -d \
+  --name postgres \
+  -e POSTGRES_PASSWORD=postgres \
+  -e POSTGRES_USER=postgres \
+  -p 5432:5432 \
+  postgres:16
+```
+
+2. **Create Test Database**:
+```bash
+docker exec postgres psql -U postgres -c "CREATE DATABASE test_db;"
+```
```

Comment:
@Isuru-rangana do we have to run the create database docker exec? The reason for asking this is that, we should just be able to do the following and do the development and deployment quite easily.